### DISCUSSION TITLE
In both multicast query and multicast response messages, the Recursion Available bit MUST be zero on transmission. (RFC6762 18.7)

### DISCUSSION THREAD

**Discussion by flotzi** [brutella/hkknx-public#225] [Q&A]
Wenn HKKNX im Docker (einer Synology Diskstation) läuft kommt in dessen Log jede Minute zwei mal der Eintrag:

02:01:45 In both multicast
query and multicast response messages, the
Recursion Available bit MUST be zero on
transmission. (RFC6762 18.7)

hkknx läuft fehlerfrei. KNX Gateway hat eine feste IP und wird per UDP auf Port 3671

kommt wohl vom dnssd @brutella 

Hat jemand eine Erklärung dafür?

Danke

**Comment by brutella** [comment]
Ja du hast Recht, dass die Meldung von [dnssd](https://github.com/brutella/dnssd/blob/665cf86dce8e8a8b4c6164e2ac038d6b667dc80b/mdns.go#L405) kommt. Ist aber nur eine Info und sollte keine Auswirkung haben. → [RFC6762 18.7](https://www.rfc-editor.org/rfc/rfc6762.html#section-18.7)

In deinem Netzwerk scheint irgendeine Software das "Recursion Available bit" falsch zu setzen.
